restcomm/restcomm.application/src/main/java/org/restcomm/connect/application/Bootstrapper.java

```diff
@@ -401,8 +402,8 @@ public void servletInitialized(SipServletContextEvent event) {
             IdentityContext identityContext = new IdentityContext(xml);
             context.setAttribute(IdentityContext.class.getName(), identityContext);
 
-            //init NumberSelectorService
-            context.setAttribute(NumberSelectorService.class.getName(), new NumberSelectorService(storage.getIncomingPhoneNumbersDao()));
+            // Initialize CoreServices
+            RestcommConnectServiceProvider.getInstance().startServices(storage);
```

Comment:
not sure if we should still publish services in servlet context for consistency purposes...

Reply:
i would not consider it a point of consistency